infra/api.tf

```diff
@@ -0,0 +1,272 @@
+    resources = [
+      data.aws_kms_alias.secretsmanager.arn
+    ]
+  }
+}
+
+data "aws_kms_alias" "secretsmanager" {
+  name = "alias/aws/secretsmanager"
+}
+
+module "ecs" {
+  source       = "git::https://gitlab.com/eternaltyro/terraform-aws-ecs.git"
+  service_name = "underpass-${var.deployment_environment}"
+  default_tags = {
+    project       = "Data_Access"
+    maintainer    = "DK"
+    documentation = "https://docs.hotosm.org/"
+    cost_center   = ""
+  }
+  container_secrets = [
+    {
+      name      = "POSTGRES_CONNECTION_PARAMS"
+      valueFrom = aws_secretsmanager_secret_version.underpass_database_credentials.arn
+    },
+    {
+      name      = "GALAXY_API_CONFIG_FILE"
+      valueFrom = aws_secretsmanager_secret_version.configfile.arn
+  }]
+  container_envvars = [
+    {
+      name  = "FORWARDED_ALLOW_IPS"
+      value = "*"
+    },
+    {
+      name  = "SENTRY_ENVIRONMENT"
+      value = var.deployment_environment
+    },
+    {
+      name  = "SENTRY_RELEASE"
+      value = lookup(var.sentry, "app_release", "galaxy-api@vDEFAULT")
+    }
+  ]
+  container_settings = {
+    service_name = "underpass-${var.deployment_environment}"
+    app_port     = 8000
+    image_url    = "ghcr.io/hotosm/underpass"
+    image_tag    = "ci"
+  }
+  container_cpu_architecture = "X86_64"
+  # container_commands = [
+  #   "",
+
+  # ]
+  container_capacity = {
+    cpu       = 256
+    memory_mb = 512
+  }
+  tasks_count = {
+    desired_count   = 1
+    min_healthy_pct = 100
+    max_pct         = 200
+  }
+  log_configuration = {
+    logdriver = "awslogs"
+    options = {
+      awslogs-group         = "underpass"
+      awslogs-region        = "us-east-1"
+      awslogs-stream-prefix = "underpass-"
+    }
+  }
+  scaling_target_values = {
+    cpu_pct             = 85
+    memory_pct          = 85
+    request_count       = 50
+    container_min_count = 1
+    container_max_count = 5
+  }
+  service_subnets          = module.vpc.public_subnets
+  aws_vpc_id               = module.vpc.vpc_id
+  service_security_groups  = [module.vpc.default_security_group_id]        // update for update
+  alb_security_group       = module.alb.load_balancer_app_security_group //public or private
+  ecs_cluster_arn          = aws_ecs_cluster.underpass.arn
+  ecs_cluster_name         = aws_ecs_cluster.underpass.name
+  task_role_arn            = aws_iam_role.underpass.arn
+  load_balancer_enabled    = true
+  load_balancer_arn_suffix = module.alb.load_balancer_arn_suffix
+  target_group_arn_suffix  = module.alb.target_group_arn_suffix
+  target_group_arn         = module.alb.target_group_arn
+  efs_enabled              = false
+  efs_settings = {
+    file_system_id     = "string"
+    root_directory     = "string"
+    access_point_id    = "string"
+    transit_encryption = "DISABLED"
+    iam_authz          = "DISABLED"
+  }
+}
+
+data "aws_acm_certificate" "wildcard" {
+  domain      = "hotosm.org"
+  statuses    = ["ISSUED"]
+  most_recent = true
```

Comment:
Please add additional line for reliability:

```
key_types = ["RSA_2048", "RSA_3072", "RSA_4096", "EC_prime256v1", "EC_secp384r1", "EC_secp521r1"]
```

AWS API by default only returns RSA certs unless you give it a list of cert types to consider. Granted we have RSA cert for our home domain. But this prepares for future.